cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgsfourleptons.json

```diff
@@ -0,0 +1,92 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan",
+        "orcid": "0000-0002-4485-2972"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "cc",
+        "py",
+        "zip"
+      ],
+      "number_files": 4,
+      "size": 999
+    },
+    "doi": "10.7483/OPENDATA.CMS.TODO",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:999",
+        "size": 999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/TODO/HiggsToFourLeptonsNanoAODOutreachAnalysis-1.0.zip"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12360",
+    "run_period": [
+      "Run2012A",
+      "Run2012B"
+    ],
+    "source_code_repository": {
+      "url": "www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis"
+    },
+    "system_details": {
+      "description": "This code can be run with a plain ROOT installation, go to <a href=\"https://root.cern\">root.cern</a> for instructions how to install the software.",
```

Comment:
Here the HTML `<a href=\"https://root.cern\">root.cern</a>` does not render properly. It is OK in the usage description but not here. @tiborsimko Could you fix that?

Reply:
I suggest removing `go to <a href="https://root.cern">root.cern</a> for instructions how to install the software` from "system_details" as the same phrase with the link comes two lines after:

![image](https://user-images.githubusercontent.com/5859918/146203767-e80ad708-7916-4cc2-b549-6556ea17834b.png)

Reply:
Removed the description field.